lib/Doctrine/ORM/Persisters/Collection/ManyToManyPersister.php

```diff
@@ -399,7 +399,7 @@ protected function getDeleteSQL(PersistentCollection $collection)
     /**
      * Internal note: Order of the parameters must be the same as the order of the columns in getDeleteSql.
      *
-     * @return mixed[]
+     * @return list<mixed>
```

Comment:
Should this be duplicated because of the `list` part?

Reply:
list is now an alias of array in phpstorm. Not sure in VS Code